### PR TITLE
Emails: use https for links and images

### PR DESCRIPTION
**Pull request by perrinjerome**
**What this PR does / why we need it**:

Changes email templates to use https instead of http.

There's an image on emails, http://grafana.org/assets/img/logo_new_transparent_200x48.png which redirects to https://grafana.com/assets/img/logo_new_transparent_200x48.png . I don't think there's a good reason for this image to be http in the first place. Most email services (such as gmail) use an https proxy to serve images, but self hosted web mails do not and this cause mixed content errors while viewing emails.

There's also a link to <a href="http://docs.grafana.org/guides/gettingstarted/">Getting Started</a>  which should not be http for extra security.

**Which issue(s) this PR fixes**:

( I did not create an issue, but I can do if you believe it's needed )

### CODE REVIEW

**Comment by aknuds1** [comment]
Please sign the CLA.

**Comment by perrinjerome** [comment]
@aknuds1 I signed CLA, thanks

**Comment by aknuds1** [comment]
@perrinjerome Do you mind if I merge with master and push to your branch?

**Comment by perrinjerome** [comment]
@aknuds1 no problem, go ahead. Thanks !

**Comment by aknuds1** [comment]
That seemed to help.

**Comment by aknuds1** [comment]
I'm checking with colleagues if these changes are correct BTW.

File: emails/templates/welcome_on_signup.html [hunk 1]
```diff
@@ -29,7 +29,7 @@ <h4>Hi [[.Name]],</h4>
 				<tr>
 					<td class="center">
 						<p>
-							If you are new to Grafana please read the <a href="http://docs.grafana.org/guides/gettingstarted/">Getting Started</a> guide.
+							If you are new to Grafana please read the <a href="https://docs.grafana.org/guides/gettingstarted/">Getting Started</a> guide.
```
Comment (papagian) [inline emails/templates/welcome_on_signup.html:32]:
```suggestion
							If you are new to Grafana please read the <a href="https://docs.grafana.com/guides/gettingstarted/">Getting Started</a> guide.
```

**Review by papagian — CHANGES_REQUESTED** [review]
Thank you for contributing! I have a small suggestion.

**Comment by perrinjerome** [comment]
Thanks @papagian I applied the change you suggested.

I did not notice there was another redirect, I just changed http to https and saw "it seems to work", so let's suggest using https here as well.

BTW, the https://docs.grafana.com/guides/gettingstarted/ also redirects (with status code 301 and location header) to https://grafana.com/docs/grafana/latest/guides/gettingstarted/ which in turn redirects (it's not a Location header but it's in the html ) to https://grafana.com/docs/grafana/latest/getting-started/getting-started/

**Review by sakjur — APPROVED** [review]
😄 This looks great!

> BTW, the https://docs.grafana.com/guides/gettingstarted/ also redirects (with status code 301 and location header) to https://grafana.com/docs/grafana/latest/guides/gettingstarted/ which in turn redirects (it's not a Location header but it's in the html ) to https://grafana.com/docs/grafana/latest/getting-started/getting-started/

Thank you for letting us know 👍 Let's keep it as-is for now and I'll let our website team know about this so that they can tell us if we should keep the original URL and fix the first redirect or change it to the final URL in that chain.

**Review by daniellee — APPROVED** [review]
LGTM

I tested with gmail and worked for me. HTML emails are their own weird world but I couldn't find anything saying that changing to https would affect anything. Seems to be the opposite in that there is a smaller chance of it getting stuck in spam filters.

**Review by papagian — APPROVED** [review]

**Review by aknuds1 — APPROVED** [review]

**Comment by aknuds1** [comment]
Thank you for contributing to Grafana!

**Comment by perrinjerome** [comment]
Thanks everybody ( and also to stale bot 😉 )

**Pull request merged**